samples/java_layout/src/main/res/values/strings.xml

```diff
@@ -58,4 +58,5 @@
     <string name="filter_view_universal_link">Java Sample Universal Scheme Links</string>
     <string name="error_text_input_field_blank">This field cannot be blank</string>
     <string name="error_text_input_field_empty">This field cannot be empty</string>
+    <string name="error_number_input_field_small">The value must be greater than %1$s</string>
```

Comment:
Shouldn't it be greater than 0? 1 should be allowed.?

Reply:
Updated error message